### PR TITLE
Create bumblebee.ms

### PR DESCRIPTION
**Pull request by jaikev**

### CODE REVIEW

**Comment by jaikev** [comment]
Angel please check this

File: CommandHelper/LocalPackages/commands/buffs/bumblebee.ms [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODc0OTg2Mjg2MDgsInByb2ZpbGVJZCI6IjU1MGIwM2NkZDllOTRjODliNGVhNDRlYWQ0ODljZTBlIiwicHJvZmlsZU5hbWUiOiJEX1N1cGVyR2lybF9DIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS8wZGRmZDYzOWRhMTc0MTJiZDg4MTJlM2UxYmQ4ZWQ2YzIyZjZiYjk2NWFlYmExMjkxMWQwNThkZTI0YiJ9fX0=',
+            'trail': '',
+            'op': 'false',
+            'speech': There are no Decepticons in the area Optimus.",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/bumblebee.ms]: missing the 1st "

**Comment by malonnnn** [comment]
bumblebee is ambiguous, please be more specific on which hero this is

**Comment by ViceTOP** [comment]
It can be A) Bumblebee the robot or B) Bumblebee DC superhero

File: CommandHelper/LocalPackages/commands/buffs/bumblebee.ms [hunk 2]
```diff
@@ -0,0 +1,25 @@
+@plr = player()
+@hero = associative_array(
+            'name': '§eBumblebee(DC)',
+            'class': 'bumblebee(dc)',
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/bumblebee.ms]: paranthesis are not allowed in the classname

**Comment by malonnnn** [comment]
the file name needs to be bumblebeedc.ms

**Pull request closed without merging**